toolboxes/core/vector_td.h

```diff
@@ -55,6 +56,14 @@ namespace Gadgetron {
             return vec[i];
         }
 
+        __inline__ __host__ __device__ T& operator()(size_t i) {
```

Comment:
Why do we need this? operator[] does the exact same thing.

Reply:
Had some code to use vector_td and using ...() already.